docs/API.rst

```diff
@@ -1,14 +1,34 @@
 API
 ===
 
-Note: This section provides a comprehensive overview of all methods and classes avaliable in PyOpenMS-Viz. This content is generated automatically using sphinx autosummary and autodoc based on the python documentation.
+Note: This section provides a comprehensive overview of all methods and classes available in PyOpenMS-Viz. This content is generated automatically using Sphinx autosummary and autodoc based on the Python documentation.
+
+Configuration Classes
+********************
+
+.. currentmodule:: pyopenms_viz._config
+
+.. autosummary::
+   :nosignatures:
+   :toctree: generated/
+   :template: class.rst
+
+   BaseConfig
+   LegendConfig
+   BasePlotConfig
+   LineConfig
+   VLineConfig
+   ScatterConfig
+   ChromatogramConfig
+   MobilogramConfig
+   SpectrumConfig
+   PeakMapConfig
```

Comment:
I would put this at the bottom of the API documentation, does not need to be this prevalent.